kolibri/core/assets/src/views/AttemptLogItem.vue

```diff
@@ -49,7 +49,7 @@
         :value="attemptLog.num_coach_contents || 0"
         :isTopic="false"
       />
-    </div>
+    </template>
```

Comment:
Changing this from a `div` to a `template` helps with the positioning of the icon because the `template` doesn't show up in the DOM, correct?

Reply:
Yes, the div was adding in an extra DOM node that messed with the layout styling that was being used.